django/contrib/gis/db/backends/spatialite/operations.py

```diff
@@ -31,6 +31,13 @@ def as_sql(self, connection, lookup, template_params, sql_params):
         return super().as_sql(connection, lookup, template_params, sql_params)
 
 
+class SpatialiteNullCheckOperator(SpatialOperator):
+    def as_sql(self, connection, lookup, template_params, sql_params):
+        sql_template = '%s > 0' % (self.sql_template or lookup.sql_template or self.default_template)
```

Comment:
Probably it could be reworked this way:
```
sql, params = super().as_sql(connection, lookup, template_params, sql_params)
return '%s > 0' % sql, params
```